absl/container/inlined_vector.h

```diff
@@ -493,7 +493,7 @@ class InlinedVector {
   //
   // NOTE: as a result of calling this overload, `other` is left in a valid but
   // unspecified state.
-  InlinedVector& operator=(InlinedVector&& other) {
+  InlinedVector& operator=(InlinedVector&& other) noexcept {
```

Comment:
This is not a move constructor. Is this intended?

Reply:
Reading through the patch it seems it was added to other move assignment operators as well. Never mind.

But I wonder if just adding noexcept is the right thing to do. This means if any operation inside the move constructor throws, it will terminate the program.
Generally, containers use `std::move_if_noexcept` to use the copy constructor in case the move constructor throws.
And then they use `noexcept( expression that determines if internal operations may throw )` to make the constructor conditionally `noexcept`